### PR TITLE
Update research.md

### PR DESCRIPTION
**Pull request by idk3**

### CODE REVIEW

File: research.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Lots more hacks built using our API can be found at the [Perspective Hacks Galle
  
 ## Research Contributions
 
+* [Toxicity Detection: Does Context Really Matter?
+](https://www.aclweb.org/anthology/2020.acl-main.396/) studies the importance of  context, and when as well as to what extent it can change the perceived toxicity of posts, finding that context can both amplify or mitigate the perceived toxicity of posts, and moreover for a subset of the data that including context can even reverse the toxicity label.
```
Comment (sorensenjs) [inline research.md]: weird new line inside [].

File: research.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ Lots more hacks built using our API can be found at the [Perspective Hacks Galle
...
+* [Classifying Constructive Comments](https://arxiv.org/abs/2004.05476) introduces a new dataset of constructive comments, defines a taxonomy of characteristics of constructiveness, and provides models for constructiveness trained on this dataset.
```
Comment (sorensenjs) [inline research.md]: This is a preprint - but I don't know if we can say much about where it's going to be published, maybe just (preprint)

**Review by sorensenjs — CHANGES_REQUESTED** [review]

File: research.md (hunk 1 above)
Comment (idk3) [reply research.md]: Weird, not sure how that happened. Doesn't appear to affect rendering, but fixed it.

File: research.md (hunk 2 above)
Comment (idk3) [reply research.md]: Appended "(preprint)" to it for now.

**Comment by idk3** [comment]
Ready for re-review :)

**Pull request merged**